skbio/stats/composition.py

```diff
@@ -2058,3 +2060,93 @@ def dirmult_ttest(
+        The formula specifying the model
+    data : array_like
+        The data for the model. data must define __getitem__ with the keys in
+        the formula terms args and kwargs are passed on to the model
+        instantiation. E.g., a numpy structured or rec array, a dictionary,
+        or a pandas DataFrame.
+    groups : str
+        The column name in data that identifies the grouping variable
+    reml : bool
+        If true, fit according to the REML likelihood, else fit the standard
+        likelihood using ML.
+    method : str
+        Optimization method. Can be a scipy.optimize method name, or a list of such
+        names to be tried in sequence.
+        See https://docs.scipy.org/doc/scipy/tutorial/optimize.html for all options.
+    fit_kwargs : dict
+        Keyword arguments to pass to the model fit function.
+        See ``statsmodels.regression.mixed_linear_model.MixedLM.fit``
+    **kwargs
+        Additional keyword arguments to pass to the mixedlm function only.
+
+    Returns
+    -------
+    results : statsmodels.regression.mixed_linear_model.MixedLMResults
+        The fitted model results.
+        See ``statsmodels.regression.mixed_linear_model.MixedLMResults``
+
+    See Also
+    --------
+    statsmodels.formula.api.mixedlm
+    statsmodels.regression.mixed_linear_model.MixedLM
+
+    Notes
+    -----
+    pass
+
+    Examples
+    --------
+    >>> data = pd.DataFrame([[1, 0, 10.2], [1, 1, 12.5],
+    ...                     [1, 7, 14.8], [1, 14, 16.1]],
+    ...                     index=[0, 1, 2, 3],
+    ...                     columns=["patient_id", "time_point",
+    ...                             "response"])
+    >>> result = dirmult_lme("response ~ time_point", groups="patient_id", data=data)
+    >>> print(result.summary()) # doctest: +SKIP
+              Mixed Linear Model Regression Results
+    =========================================================
+    Model:             MixedLM  Dependent Variable:  response
+    No. Observations:  4        Method:              REML
+    No. Groups:        1        Scale:               1.4264
+    Min. group size:   4        Log-Likelihood:      -6.3004
+    Max. group size:   4        Converged:           Yes
+    Mean group size:   4.0
+    ---------------------------------------------------------
+               Coef.     Std.Err.     z   P>|z| [0.025 0.975]
+    ---------------------------------------------------------
+    Intercept  11.345         1.459 7.777 0.000  8.486 14.204
+    time_point  0.374         0.107 3.497 0.000  0.164  0.583
+    Group Var   1.426 160301149.746
+    =========================================================
+    <BLANKLINE>
+
+    """
+
+    # Fit the Dirichlet Multinomial linear mixed effects model
+    model = MixedLM.from_formula(formula=formula, data=data, groups=groups, **kwargs)
+    if fit_kwargs is None:
+        fit_kwargs = {}
+    results = model.fit(reml=reml, method=method, **fit_kwargs)
```

Comment:
I think something was lost in translation here. 

There should be a double nested loop over [posterior draws](https://github.com/scikit-bio/scikit-bio/blob/main/skbio/stats/composition.py#L2003-L2009) and [features](https://github.com/scikit-bio/scikit-bio/blob/main/skbio/stats/composition.py#L2011-L2016) -- it should be almost an exact mirror of the dirmult_ttest, except with the ttest swapped out with the MixedLM function